src/common/core/users/serialization.js

```diff
@@ -1031,6 +1061,8 @@ define(['common/util/assert', 'blob/BlobConfig'], function (ASSERT, BlobConfig)
                     core.setPointer(node, keys[i], null);
                 } else if (nodes[target] && toRemoveGuids.indexOf(target) === -1) {
                     core.setPointer(node, keys[i], nodes[target]);
+                } else if (isCombinedId(target)) {
+                    core.setPointer(node, keys[i], getCombinedTarget(target));
                 } else {
                     console.log('error handling needed???!!!???');
```

Comment:
I was checking the logger, but either the public API should be changed, or some global logger should be created.
On the other hand the given else branch is considered (again) a never run into branch, that is why I left it that way.